compiler/rustc_target/src/spec/windows_uwp_gnu_base.rs

```diff
@@ -24,7 +24,7 @@ pub fn opts() -> TargetOptions {
     TargetOptions {
         abi: "uwp".into(),
         vendor: "uwp".into(),
-        executables: false,
+        executables: false, // FIXME?
```

Comment:
cc @ChrisDenton @chouquette
Is this intentional?
Are UWP targets unable to generate executables with mingw specifically?
This flag is true on MSVC UWP targets.

Reply:
At the time of writing I didn't know of a way to generate an executable from native code when targeting UWP but:
- This might be a false assumption coming from WinRT times
- This might not be true anymore, or at all

By all means, if switching this to `true` is all that's required to generate an executable, please do!

Reply:
I would check with @mati865 as the resident mingw expert. I would note that these are tier 3 targets so it may not necessarily be problem if there are extra steps required to make the binary actually work. We also have XP as a tier 3 which won't work out of the box.

Reply:
Definitely this should be set to true.

Reply:
Changed to `executables: true`.